moduleframework/common.py

```diff
@@ -42,24 +42,82 @@
 from avocado.utils import process
 from moduleframework.mtfexceptions import ModuleFrameworkException, ConfigExc, CmdExc, DefaultConfigExc
 
+
+def print_info(*args):
+    """
+    Print information from the expected stdout and
+    stderr files from the native test scope.
+
+    See `Test log, stdout and stderr in native Avocado modules
+    <https://avocado-framework.readthedocs.io/en/latest/WritingTests.html
+    #test-log-stdout-and-stderr-in-native-avocado-modules>`_ for more information.
+
+    :param args: object
+    :return: None
+    """
+    for arg in args:
+        print(arg, file=sys.stderr)
+
+
+def print_debug(*args):
+    """
+    Print information from the expected stdout and
+    stderr files from the native test scope if
+    the **DEBUG** envvar is set to True.
+
+    See `Test log, stdout and stderr in native Avocado modules
+    <https://avocado-framework.readthedocs.io/en/latest/WritingTests.html
+    #test-log-stdout-and-stderr-in-native-avocado-modules>`_ for more information.
+
+    :param args: object
+    :return: None
+    """
+    if is_debug():
+        print_info(*args)
+
+
+
+class MTFConfParser(object):
+    configname = "mtf.yaml"
+    source_cfg_file = os.path.join(os.path.dirname(__file__), configname)
+    default_cfg_file = os.path.join("/etc", configname)
```

Comment:
Add `/etc` at the beginning of the file common.py. There is already set of constants.